helm-chart/templates/deployment.yaml

```diff
@@ -38,7 +38,11 @@ spec:
         - configMapRef:
             name: {{ include "json-rpc-relay.fullname" . }}
         - secretRef:
+        {{- if .Values.existingSecret }}
+            name: {{ .Values.existingSecret }}
+        {{- else }}
             name: {{ include "json-rpc-relay.fullname" . }}  
+        {{- end }}
```

Comment:
Is this really the best approach to take? We use a more simple, but customizable approach on the mirror node where we have a single [envFrom](https://github.com/hashgraph/hedera-mirror-node/blob/37a98829aa97872421435a3c7d6c92df603766de/charts/hedera-mirror-web3/values.yaml#L42) that users can add custom secrets or configmaps with templating support. So relay could do the same and have the existing secret and configmap moved to values.yaml and allow you to override it for your purposes.